readthedocs/projects/forms.py

```diff
@@ -444,23 +445,46 @@ def save(self):
 
 class WebHookForm(forms.ModelForm):
 
-    """Project webhook form."""
+    project = forms.CharField(widget=forms.HiddenInput(), required=False)
+
+    class Meta:
+        model = WebHook
+        fields = ['project', 'url', 'events', 'payload', 'secret']
+        widgets = {
+            'events': forms.CheckboxSelectMultiple,
```

Comment:
I think this is better than the list, and we don't have that many events. I wanted to show something similar to what github does (the name and the description)
![Screenshot 2021-09-28 at 15-07-07 Webhook · https requires io github web-hook · readthedocs readthedocs org](https://user-images.githubusercontent.com/4975310/135158172-ff83fff1-9a11-43c2-866e-58e5630c7975.png)

but looks like we would need to just manually do that, there isn't a widget that does it for us :/